client/apollo/css/src/Toggle/ToggleLF.scss

```diff
@@ -0,0 +1,41 @@
+@use "./ToggleCommon";
+
+.af-toggle {
+  --toggle-bg-color: var(--blue-200);
+  --toggle-border-color: var(--gray-700);
+  --toggle-border-radius: var(--radius-32);
+  --toggle-border-size: 2px;
+  --toggle-padding: 4px;
+  --toggle-width: 52px;
+  --toggle-height: 32px;
+  --toggle-handle-color: var(--gray-700);
+  --toggle-handle-size: 16px;
+  --toggle-handle-position: calc(var(--toggle-border-size) * 2);
+  --toggle-handle-scale: 1;
+
+  &:has(:checked) {
+    --toggle-bg-color: var(--blue-800);
+    --toggle-border-color: unset;
+
+    .af-toggle__handle {
+      --toggle-handle-scale: 1.5;
+      --toggle-handle-color: var(--white);
```

Comment:
à vérifier mais il me semble que c'est --color-white en L&F
tu peux retrouver le nom des tokens L&F ici : https://github.com/AxaFrance/design-system/blob/main/client/look-and-feel/css/src/common/tokens.scss